pyccl/covariances.py

```diff
@@ -127,3 +129,240 @@ def angular_cl_cov_cNG(cosmo, cltracer1, cltracer2, ell, tkka, fsky=1.,
 
     check(status, cosmo=cosmo_in)
     return cov
+
+
+def sigma2_B_disc(cosmo, a=None, fsky=1., p_of_k_a=None):
+    """ Returns the variance of the projected linear density field
+    over a circular disc covering a sky fraction `fsky` as a function
+    of scale factor. This is given by
+
+    .. math::
+        \\sigma^2_B(z) = \\int_0^\\infty \frac{k\\,dk}{2\\pi}
+            P_L(k,z)\\,\\left[\\frac{2J_1(k R(z))}{k R(z)}\\right]^2,
+
+    where :math:`R(z)` is the corresponding radial aperture as a
+    function of redshift. This quantity is used to compute the
+    super-sample covariance.
+
+    Args:
+        cosmo (:class:`~pyccl.core.Cosmology`): a Cosmology object.
+        a_arr (float, array_like or `None`): an array of scale factor
+            values at which to evaluate the projected variance. If
+            `None`, a default sampling will be used.
+        fsky (float): sky fraction.
+        p_of_k_a (:class:`~pyccl.pk2d.Pk2D`, str, or `None`): Linear
+            power spectrum to use. Defaults to `None`.
```

Comment:
Maybe add that, if `None`, the internal linear power spectrum is used.

Reply:
Done.